l5kit/l5kit/rasterization/rasterizer_builder.py

```diff
@@ -126,6 +126,7 @@ def build_rasterizer(cfg: dict, data_manager: DataManager) -> Rasterizer:
         raster_size_px=np.array(raster_cfg["raster_size"]),
         pixel_size_m=np.array(raster_cfg["pixel_size"]),
         center_in_raster_ratio=np.array(raster_cfg["ego_center"]),
+        origin_bottom=raster_cfg["origin_bottom"],
```

Comment:
```suggestion
        vertical_flip=raster_cfg["set_origin_to_bottom"],
```

Reply:
`origin_bottom` is unclear, what about `vertical_flip`?

Reply:
I don't think it's more clear, because both can be seen as the flip of the other, but I'm fine in changing it

Reply:
Agree, how about rename  parameter `origin_bottom` to `set_origin_to_bottom`?